code/__DEFINES/augments.dm

```diff
@@ -19,6 +19,9 @@
 #define AUGMENT_SLOT_STOMACH		"Stomach"
 #define AUGMENT_SLOT_EYES			"Eyes"
 #define AUGMENT_SLOT_TONGUE			"Tongue"
+#define AUGMENT_SLOT_FLUFF_HEAD		"Minor Head Augment"
+#define AUGMENT_SLOT_FLUFF_CHEST	"Minor Chest Augment"
+#define AUGMENT_SLOT_FLUFF_GROIN	"Minor Groin Augment"
```

Comment:
I don't like this but I also don't have any better ideas.

Reply:
I originally was going to use the implant system but it turned out using that would have been much more scuffed

I don't know of any way off adding these without it being scuffed other than adding a completely new system, which would feel like bloating character customization